scripts/issue/791/deprecate_swapr.py

```diff
@@ -0,0 +1,33 @@
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import registry, get_registry
+
+REGISTRY = get_registry()
+TECH_OPS = REGISTRY.badger_wallets.techops_multisig
+
+SWAPR_WETH = REGISTRY.sett_vaults.bdxsSwaprWeth
+WBTC_WETH = REGISTRY.sett_vaults.bdxsWbtcWeth
+BADGER_WETH = REGISTRY.sett_vaults.bdxsBadgerWeth
+IBBTC_WETH = REGISTRY.sett_vaults.bdxsIbbtcWeth
+
+
+def deprecate_polygon():
+    safe = GreatApeSafe(TECH_OPS)
+    safe.init_badger()
+
+    registry = safe.contract(REGISTRY)
```

Comment:
The function `get_registry` returns the address book, not the registryV2 contract. You should instead do:
```
registry = safe.contract(REGISTRY.registry_v2)
```